app/src/main/java/com/it3048/mobiledeviceproject/AddMeetingForm.kt

```diff
@@ -1,35 +1,38 @@
 package com.it3048.mobiledeviceproject
 
 import android.os.Bundle
-import android.util.Log
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
-import android.widget.Toast
 import androidx.fragment.app.DialogFragment
+import com.it3048.mobiledeviceproject.databinding.AddMeetingFormBinding
 import kotlinx.android.synthetic.main.add_meeting_form.view.*
 
 import com.it3048.mobiledeviceproject.dto.Meeting
 
 class AddMeetingForm: DialogFragment() {
 
     private var meeting = Meeting()
+    private var _binding: AddMeetingFormBinding? = null
+    private val binding get() = _binding!!
 
     override fun onCreateView(
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
+        _binding = AddMeetingFormBinding.inflate(inflater, container, false)
+        val view = binding.root
         // Inflate the layout for this fragment
-       var rootView: View = inflater.inflate(R.layout.add_meeting_form, container, false)
 
-        rootView.cancelButton.setOnClickListener{
+        view.cancelButton.setOnClickListener{
             dismiss()
-            clearFields(rootView)
+            clearFields(view)
```

Comment:
Used the jetpack viewbinding library and bound the root to a view that is then passed to the other functions.
Naming conventions according to:
https://kotlinlang.org/docs/coding-conventions.html#names-for-backing-properties
Process according to:
https://developer.android.com/topic/libraries/view-binding#fragments